### PR TITLE
Added Swift Data Types

### LINKED ISSUE #89: Add Swift data types
# Description

Data Types for Swift aren't presently on Code Thesaurus. This issue is to add that concept to the system.

# Requirements

* Copy `web/thesauruses/_meta/data_types.json` to the right Swift directory in `web/thesauruses/`
* Edit the data_types.json file to accurately reflect the latest major version of Swift
* Delete the `"name":` fields before your PR
* Add in `"comment":` if you wish to explain something in the code
* Add in `"not-implemented": "true"` if the concept doesn't exist in the language. You can then leave `"code":` blank.

# Claiming This Issue

Please leave a comment below so I can assign you to it. This is to prevent multiple people from working on the same issue. 

Check out [CONTRIBUTING.md](https://github.com/codethesaurus/codethesaur.us/blob/main/CONTRIBUTING.md) for more information on how to contribute. 

Check out [https://docs.codethesaur.us/thesaurus/](https://docs.codethesaur.us/thesaurus/) for information on how Code Thesaurus works and how to add to the thesauruses.

# Need Help?

Leave a comment on here with your question, or feel free to ask [@geekygirlsarah](https://twitter.com/geekygirlsarah) on Twitter. You can also email coreteam@codethesaur.us too.

# Thanks!

We really value and appreciate the additions people make to Code Thesaurus, no matter how large or small. So thank you for taking the time to work on this!

### PR DESCRIPTION
**Pull request by gildedgardenia**
## What GitHub issue does this PR apply to?

Resolves #89

## What changed and why?

Added information about data types in Swift.

## Please add screenshots

swift-data-types-page

## Checklist

- [x] I claimed any associated issue(s) and they are not someone else's
- [x] I have looked at documentation to ensure I made any revisions correctly
- [x] I tested my changes locally to ensure they work
- [ ] (If editing Django) I have added or edited any appropriate unit tests for my changes

## Any additional comments or things to be aware of while reviewing?

Swift types were somewhat hard to classify! Please let me know if something doesn't make sense!

### CODE REVIEW

File: web/thesauruses/swift/data_types.json [hunk 1]
```diff
@@ -0,0 +1,123 @@
+{
+   "meta": {
+     "language": "language_id",
```
Comment (geekygirlsarah) [inline web/thesauruses/swift/data_types.json]: Can you update the language to be `swift` and the version to be whatever the latest version to me?

File: web/thesauruses/swift/data_types.json [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+     },
+     "signed_integer_64_bit": {
+       "code": "Int64"
+     },
+     "unsigned_integer_64_bit": {
+       "code": "UInt64"
+     },
+     "signed_integer_as_object": {
+       "code": "Int",
+       "comment": "``Int`` is used in situations where the platform size may not be known. On a 32-bit platform, ``Int`` can store 32-bit integers."
```
Comment (geekygirlsarah) [inline web/thesauruses/swift/data_types.json]: I'm curious about the double backticks. There's now Markdown support, so you probably only need one of them on each side for it to render right on the site.

**Review by geekygirlsarah — CHANGES_REQUESTED** [review]
I think this is pretty good. Make the couple of small changes and I can merge it in!

File: web/thesauruses/swift/data_types.json (hunk 1 above)
Comment (gildedgardenia) [reply web/thesauruses/swift/data_types.json]: Done!

File: web/thesauruses/swift/data_types.json (hunk 2 above)
Comment (gildedgardenia) [reply web/thesauruses/swift/data_types.json]: The double backticks is a bad habit that I keep carrying over from writing RST. I switched them to single backticks. Thanks!

Comment (geekygirlsarah) [reply web/thesauruses/swift/data_types.json]: Oooh, RST. That makes sense. Also I've learned double-backticks do something special in Markdown, so I guess they're not _wrong_ but do something different than I think you were expecting.

**Comment by geekygirlsarah** [comment]
I think this should work! I'll merge it in.

Thanks again! Don't forget to sign up for Hacktoberfest if you haven't yet! You're halfway there!

**Pull request merged**